gulpfile.js

```diff
@@ -12,5 +12,6 @@ var elixir = require('laravel-elixir');
  */
 
 elixir(function(mix) {
-    mix.less('app.less');
+    mix.less('admin.less');
+    mix.sass('main.scss');
```

Comment:
I think you can probably accomplish most of the same things with Elixir (like `mix.browserify` and `mix.copy`), and it'd be nice to avoid having custom build toolchains for individual repos when we can.